builder/processor-tools/src/main/java/io/helidon/builder/processor/tools/DefaultBuilderCreatorProvider.java

```diff
@@ -400,14 +420,34 @@ protected void appendHeader(StringBuilder builder,
         builder.append(" {\n");
     }
 
+    /**
+     * Returns the copyright level header comment.
+     *
+     * @param ctx   the context
+     * @return the copyright level header
+     */
+    protected String generatedCopyrightHeaderFor(BodyContext ctx) {
+        return copyrightHeaderFor(getClass().getName());
+    }
+
     /**
      * Returns the {@code Generated} sticker to be added.
      *
      * @param ctx   the context
-     * @return the generated sticker.
+     * @return the generated sticker
      */
     protected String generatedStickerFor(BodyContext ctx) {
-        return BuilderTypeTools.generatedStickerFor(getClass().getName(), Versions.CURRENT_BUILDER_VERSION);
+        return BuilderTypeTools.generatedStickerFor(getClass().getName(), generatedVersionFor(ctx));
```

Comment:
Well, in many other places in this class, there are methods (such as `generatedStickerFor`), called directly and without `BuilderTypeTools` class specified.

Reply:
The `BuilderTypeTools` prefix on this case is necessary since this class is reusing the same method name, the one that takes `BodyContext` as the argument.